test/rules/typedef/variable-declaration-ignore-function/test.ts.lint

```diff
@@ -0,0 +1,17 @@
+// Should all pass
+var foo = function(): void {};
+let foo = function(): void {};
+const foo = function(): void {};
+
+var foo = (): void => {};
+let foo = (): void => {};
+const foo = (): void => {};
+
+class Foo {
+    foo = (): void => {};
+    foo = function(): void {};
+}
+
+// Should not throw any exception if we do decide to add types.
```

Comment:
The test lines themselves imply enough context that we can remove the redundant comments. The comments tend to get out of date.

```suggestion
```